StudCity.Frontend/src/pages/RegistrationComplete/steps/ChooseGroups.jsx

```diff
@@ -29,23 +31,25 @@ const ChooseGroups = ({ groups, setGroups }) => {
 	return (
 		<div className='h-full flex flex-col justify-center items-center'>
 			<h1 className="text-4xl text-center mb-6">Choose your first groups</h1>
-			{allGroups.length ? <div className='flex flex-wrap gap-2 items-center justify-center'>
-				{allGroups.map((group) => {
-					const isActive = groups.includes(group.id);
-					return <GroupCard key={group.id}
-						group={group}
-						isActive={isActive}
-						handleClick={handleClick}
-					/>
-				})
-				}
-				<div className='flex flex-col justify-center items-center'>
-					{groups.length !== 0 && <h2 className='my-2 text-xl'>Selected: {groups.length}</h2>}
-					<IconButton className='my-2' onClick={fetchGroups}>
-						<i className="fa-solid fa-rotate-right"></i>
-					</IconButton>
-				</div>
-			</div> : <Spinner />}
+			<Scroller>
+				{allGroups.length ? <div className='w-11/12 mx-auto py-6 grid grid-cols-2 gap-5 place-items-center'>
+					{allGroups.map((group) => {
+						const isActive = groups.includes(group.id);
```

Comment:
I would probably suggest to just wright this logic  directly for GroupCard isActive property, because there is not much logic here, and returning component directly from map just looks clearer in my opinion
But this one good as well

Reply:
@Serhiy-dotcom We can discuss this offer [here](https://meet.google.com/sas-yifd-rbj?authuser=0)